### PR TITLE
Checkov

### PR DESCRIPTION
**Pull request by ben851**
# Summary | Résumé

This will switch the (not working) TF Sec scanning check to use Checkov instead. Checkov will also automatically publish the results to the [GitHub code scanning tracker](https://github.com/cds-snc/notification-terraform/security/code-scanning?query=pr%3A910+is%3Aopen). 

I have configured Checkov to run in permissive mode so that the pipelines don't fail when there are failing tests. At the moment there are a large number of failing tests that we will have to triage and either fix, or add to the ignore list should they be deemed false alarms.

### CODE REVIEW

File: .github/workflows/terragrunt_plan_staging.yml [hunk 1]
```diff
@@ -301,4 +301,4 @@ jobs:
           comment-delete: "true"
           comment-title: "Staging: sns_to_sqs_sms_callbacks"
           github-token: "${{ secrets.GITHUB_TOKEN }}"
-          terragrunt: "true"
+          terragrunt: "true"
```
Comment (sastels) [inline .github/workflows/terragrunt_plan_staging.yml:304]: Github is happiest when there's a blank line at the end 😂

File: .gitignore [hunk 2]
```diff
@@ -6,6 +6,7 @@
 *.tfstate.*
 *.tfvars
 *.yaml
+!.checkov.yaml
```
Comment (sastels) [inline .gitignore:9]: The file has a ".yml" extension so it would probably be included anyway? Or were you intending to name with a ".yaml"

**Review by sastels — APPROVED** [review]
LGTM! Could add those blank lines to get GitHub happy and change the ".yml" suffix (or change the .gitignore) if you wise

File: .gitignore (hunk 2 above)
Comment (ben851) [reply .gitignore:9]: Whoops, meant to add both.

**Pull request merged**